crates/holochain_p2p/Cargo.toml

```diff
@@ -25,6 +25,7 @@ kitsune_p2p = { version = "0.0.44", path = "../kitsune_p2p/kitsune_p2p" }
 kitsune_p2p_types = { version = "0.0.32", path = "../kitsune_p2p/types" }
 mockall = "0.10.2"
 observability = "0.1.3"
+parking_lot = "0.10"
```

Comment:
o_O - don't see any parking lot usage additions in this particular crate...

Reply:
whoops!